resources/references/security.md

```diff
@@ -4,10 +4,30 @@
 
 This reference presents a comprehensive compilation of all security measures implemented in Shopware 6, along with instructions on how to configure them.
 
+## ACL in the Administration
+
+The Access Control List (ACL) in Shopware ensures that by default, data can only be created, read, updated, or deleted ( CRUD), once the user has specific privileges for a module. [ACL in the Administration](../concepts/framework/architecture/administration-concept.md#acl-in-the-administration)
```

Comment:
```suggestion
The Access Control Layer (ACL) in Shopware ensures that by default, data can only be created, read, updated, or deleted (CRUD), once the user has specific privileges for a module. [ACL in the Administration](../concepts/framework/architecture/administration-concept.md#acl-in-the-administration)
```

Reply:
As discussed in slack we leave it with "List" - I will change the space as a new commit and resolve the conversation.